test/TextToBinary.PipeStorage.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2016 Google
```

Comment:
I'm not a license lawyer. So I'd think @dneto0 should check and make sure this is correct.